functions/index.js

```
@@ -0,0 +1,345 @@
+    console.log('실패:', response.failureCount, '건');
+    
+    // 실패한 토큰들 로그 출력
+    if (response.failureCount > 0) {
+      response.responses.forEach((resp, idx) => {
+        if (!resp.success) {
+          const errorCode = resp.error && resp.error.code ? resp.error.code : 'unknown';
+          const errorMessage = resp.error && resp.error.message ? resp.error.message : 'unknown error';
+          console.error(`토큰 ${idx} 전송 실패:`, errorCode, errorMessage);
+          
+          // 만료된 토큰이나 잘못된 토큰인 경우 로그 출력
+          if (errorCode === 'messaging/invalid-registration-token' ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)